source/manual/govuk-notify.html.md

```diff
@@ -0,0 +1,73 @@
+---
+owner_slack: "#govuk-developers"
+title: How we use GOV.UK Notify
+section: Emails
+type: learn
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-04-29
+review_in: 6 months
+---
+
+[GOV.UK Notify][notify] is as Government-as-a-Platform service that allows
+clients of their API to send emails, text messages and letters. We use GOV.UK
+Notify to send emails to users, both citizens and publishers. Historically
+we've also used AWS SES to send emails, but that's being phased out in favour
+of GOV.UK Notify.
+
+[notify]: https://www.notifications.service.gov.uk/
+
+We have two main services on GOV.UK Notify (six in total as we have a service
+for each environment):
+
+- **GOV.UK Emails**
+
+  This service is used by Email Alert API only. It's used to send citizen-facing
+  email updates about pieces of content on GOV.UK. It's our biggest sender of
+  emails and regularly exceeds one million emails per day.
+
+- **GOV.UK Publishing**
+
+  This service is used by our publishing applications. It's used to send
+  publisher-facing emails. The type of email can vary a lot, but a typical
+  example might be to receive an email when a scheduled document on GOV.UK has
+  been automatically published.
+
+_In the future we may set up a new 'GOV.UK Public' service which is used to
+send citizen-facing emails which haven't gone through Email Alert API._
```

Comment:
I'd remove this line - it's missing the 'why', and is an unnecessary complication until it's implemented.

Reply:
I added this because it's come up a few times (and more recently with the Coronavirus services) about which GOV.UK Notify account to use when sending public facing emails and not using Email Alert API. It's likely we're going to need to create an account for this very soon (@kevindew knows more at the moment) but it's not there yet, which is why Ieft the line this like.

I could remove it, but I wanted it there as a kind of reminder to the next person who reviews it...

Reply:
Oh ok, I was missing that context. On the basis that it becomes reality soon, I'm happy for this to be in the docs.